tftpsync/susemanager-tftpsync-recv/add.wsgi

```diff
@@ -16,74 +17,93 @@ import os
 import re
 import logging
 import logging.handlers
+
+# pylint: disable=deprecated-module
 import cgi
 import tempfile
+
 try:
     from cStringIO import OutputType
-except:
+except ImportError:
     from io import StringIO as OutputType
 from io import BytesIO
 from spacewalk.common.rhnConfig import CFG, initCFG
 
 initCFG("tftpsync")
 
 # create logger
-logger = logging.getLogger('tftpsync_add')
+logger = logging.getLogger("tftpsync_add")
 logger.setLevel(logging.INFO)
 
 # create RotatingFileHandler handler and set level to INFO
-ch = logging.handlers.RotatingFileHandler("/var/log/tftpsync/tftpsync.log",
-                                          mode='a', maxBytes=1048576, backupCount=10)
+ch = logging.handlers.RotatingFileHandler(
+    "/var/log/tftpsync/tftpsync.log", mode="a", maxBytes=1048576, backupCount=10
+)
 ch.setLevel(logging.INFO)
 
 # create formatter
-formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
 
 # add formatter to ch
 ch.setFormatter(formatter)
 
 # add ch to logger
 logger.addHandler(ch)
 
+
 class TftpFieldStorage(cgi.FieldStorage):
 
-    def make_file(self, binary=None):
+    def make_file(self, **_):
```

Comment:
I'd prefer to stick to our style guide and handle unused arguments explicitly (as explained at the bottom of [2.1](https://google.github.io/styleguide/pyguide.html#21-lint)

Reply:
So that means
```suggestion
    def make_file(self, **kwargs):
        del kwargs
```

Reply:
This should be done, now.